help_to_heat/frontdoor/schemas.py

```diff
@@ -598,34 +598,81 @@
         "value": "No loft",
     },
 )
-multichoice_options = (
+multichoice_options_agreement = (
```

Comment:
Minor, but I think it's better to start the name with the most pertinent information. In this case, I think `satisfaction_multichoice_options`, `agreement_multichoice_options` and `service_usage_multichoice_options` make intent/distinctions a bit clearer.

Reply:
Changed these variable names 😄